src/app/contact-page/contact-page.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container w-100 h-100">
+    <div class="text-center mt-5 row justify-content-center">
+        <div>
+            <h1>{{'contact-us.title' | translate}}</h1>
+            <div>
+              <p>{{'contact-us.description' | translate}}</p>
+              <div class="text-left">
+                <span>{{'contact-us.form' | translate}} </span>
+                <a href="info/feedback" class="ml-4">{{'contact-us.feedback' | translate}}</a>
+              </div>
+              <div class="text-left">
+                <span>{{'contact-us.email' | translate}}</span>
+                <a href="mailto:marketa.trykarova@tul.cz" class="email-text"
```

Comment:
I think this email should be loaded from the cfg also, check the: https://github.com/dataquest-dev/dspace-angular/blob/dtq-dev/src/app/bitstream-page/clarin-license-agreement-page/clarin-license-agreement-page.component.html#L24